include/zephyr/dt-bindings/clock/mcux_lpc_syscon_clock.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-22, NXP
+ * Copyright (c) 2020-2022, NXP
```

Comment:
We can delete the `(c)` from the Copyright

Reply:
Deleted the (c), thanks.